src/Features/Core/Portable/ChangeSignature/ChangeSignatureTelemetryLogger.cs

```diff
@@ -0,0 +1,213 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using Microsoft.CodeAnalysis.Internal.Log;
+
+namespace Microsoft.CodeAnalysis
+{
+    internal class ChangeSignatureLogger
+    {
+        private const string Maximum = nameof(Maximum);
+        private const string Minimum = nameof(Minimum);
+        private const string Mean = nameof(Mean);
+
+        private static readonly LogAggregator s_logAggregator = new LogAggregator();
+        private static readonly StatisticLogAggregator s_statisticLogAggregator = new StatisticLogAggregator();
+        private static readonly HistogramLogAggregator s_histogramLogAggregator = new HistogramLogAggregator(bucketSize: 50, maxBucketValue: 1000);
```

Comment:
@genlu I copied your values for this, but am not sure what I'm doing. Can you look at my statistic/histogram telemetry and let me know if it makes sense?

Reply:
Moved this to https://github.com/dotnet/roslyn/pull/43504